pkgs/tools/misc/bonk/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "bonk";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "elliot40404";
+    repo = "bonk";
+    rev = "v${version}";
+    sha256 = "sha256-Y6Hia+B7kIvdvpuZwWGJBsn+pOBmMynXai4KWkNs4ck=";
+  };
+
+  cargoHash = "sha256-XphSjB49zFB3zXYpdjjcVRdTAW2Bvg91aZkxDLvFy3M=";
+
+  meta = with lib; {
```

Comment:
Could you please add `meta.mainProgram` in here ? With attributes sorted alphabetically and if you could remove the `with lib;` as well.

Reply:
I am not sure why we need meta.mainProgram when the executable name is same as the pname?

Reply:
Haha, see this https://github.com/NixOS/nixpkgs/pull/246386